index.html

```diff
@@ -1,113 +1,226 @@
+        <button class="card__showanswer">Show answer</button>
+        <p class="card__answer card__answer--hidden">
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Esse amet
+          eligendi dignissimos nam, consectetur repellat? Ipsam, totam porro.
+          Placeat, quae cupiditate deleniti veritatis in autem expedita
+          voluptate vero nisi hic?
+        </p>
+        <ul class="tags">
+          <li class="tags__item">BEM</li>
+          <li class="tags__item">CSS</li>
+          <li class="tags__item">Structure</li>
+        </ul>
+      </section>
 
-        <section class="card">
-            <button class="card__bookmark" aria-label="bookmark"></button>
-            <h2 class="card__question">What is BEM?</h2>
-            <button class="card__showanswer">Show answer</button>
-            <p class="card__answer card__answer--hidden">
-                Lorem, ipsum dolor sit amet consectetur adipisicing elit. Esse amet
-                eligendi dignissimos nam, consectetur repellat? Ipsam, totam porro.
-                Placeat, quae cupiditate deleniti veritatis in autem expedita
-                voluptate vero nisi hic?
-            </p>
-            <ul class="tags">
-                <li class="tags__item">BEM</li>
-                <li class="tags__item">CSS</li>
-                <li class="tags__item">Structure</li>
-            </ul>
-        </section>
+      <section class="card">
+        <button class="card__bookmark"></button>
+        <h2 class="card__question">What is BEM?</h2>
+        <button class="card__showanswer">Show answer</button>
+        <p class="card__answer card__answer--hidden">
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Esse amet
+          eligendi dignissimos nam, consectetur repellat? Ipsam, totam porro.
+          Placeat, quae cupiditate deleniti veritatis in autem expedita
+          voluptate vero nisi hic?
+        </p>
+        <ul class="tags">
+          <li class="tags__item">BEM</li>
+          <li class="tags__item">CSS</li>
+          <li class="tags__item">Structure</li>
+        </ul>
+      </section>
+    </main>
+    <main class="main main--hidden create">
+      <form class="form">
+        <label class="form__label" for="question">What's your question?</label>
+        <textarea
+          class="form__textarea"
+          id="question"
+          name="question"
+          rows="2"
+          cols="35"
+          placeholder="Enter your question here..."
+        ></textarea>
+
+        <label class="form__label" for="answer">What is the answer?</label>
+        <textarea
+          class="form__textarea"
+          id="answer"
+          name="answer"
+          rows="7"
+          cols="35"
+          placeholder="Enter your answer here..."
+        ></textarea>
+
+        <label class="form__label" for="tags">Tags</label>
+        <textarea
+          class="form__textarea"
+          id="question"
```

Comment:
IDs need to be unique. There are two `id="question"`

Reply:
Thanks, removed them. They weren't used anyway 😄